packages/hoppscotch-common/src/components/http/TabHead.vue

```diff
@@ -58,6 +59,19 @@
               }
             "
           />
+          <HoppSmartItem
+            ref="shareRequestAction"
+            :icon="IconShare2"
+            :label="t('tab.share_tab_request')"
+            :shortcut="['S']"
+            :new="true"
```

Comment:
`HoppSmartItem` doesn't seem to accept a `new` prop.

Reply:
fixed in this [commit](https://github.com/hoppscotch/hoppscotch/pull/3779/commits/6a4b7e6e0ed90094b0979a727061f6a56918ee2a)